api/api_test.go

```diff
@@ -33,6 +33,18 @@ var issuesRemovableProtected = hstspreload.Issues{
 	},
 }
 
+var issuesRemovableSubdomain = hstspreload.Issues{
+	Errors: []hstspreload.Issue{
+		{Code: "server.removable.subdomain"},
```

Comment:
Nit: we use `server.remove.*` already.
```suggestion
		{Code: "server.remove.subdomain"},
```

Reply:
I think that `server.removable` is correct for these, as they are error codes for the Removable API endpoint, not the Remove API endpoint. (E.g. there's both a `server.removable.protected` and a `server.remove.protected` error code, depending on whether the request was made to /api/v2/remove or /api/v2/removable.)